### ISSUE TITLE
Effect of loss components on each branch

### ISSUE THREAD

**Issue by mkhoshle** [Arthur151/ROMP#434]
Hi @Arthur151. 
Several questions:

- In the ROMP paper, I see that you have defined weighted loss. Looking into the `learnable_loss` function it seems that the total loss is just the summation of the component's losses and I do not see any weights multiplied. 
 If some tasks are more critical than others, their losses may be weighted differently in the overall loss function. Is this the case in ROMP training? 

- Also looking at the architecture it seems that there are three heads for calculating different parameters (center heatmap, smpl map, and camera map). Are you considering these tasks to be interdependent or share some common features? 

- Are the losses for different tasks affecting the weights of each layer independently? For example, do the loss components for smpl branch affect the weights for the center map branch or vice versa?

I would greatly appreciate your feedback on this.
Thanks
